src/mokka/src/main/java/pl/hycom/mokka/service/openapi/OpenAPIToStubMappingListWrapperConverter.java

```diff
@@ -0,0 +1,192 @@
+package pl.hycom.mokka.service.openapi;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.http.ResponseDefinition;
+import com.github.tomakehurst.wiremock.matching.RequestPattern;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import io.swagger.v3.oas.models.*;
+import io.swagger.v3.oas.models.media.MediaType;
+import io.swagger.v3.oas.models.responses.ApiResponse;
+import io.swagger.v3.oas.models.responses.ApiResponses;
+import org.json.JSONObject;
+import org.json.XML;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+import pl.hycom.mokka.service.openapi.pojo.StubMappingListWrapper;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+@Component
+public class OpenAPIToStubMappingListWrapperConverter implements Converter<OpenAPI, StubMappingListWrapper> {
+
+    private static final String HTTP_STATUS_OK_CODE = "200";
+
+    private static final String CONTENT_TYPE_KEY = "Content-Type";
+
+    private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
+
+    private static final String APPLICATION_XML_CONTENT_TYPE = "application/xml";
+
+    @Autowired
```

Comment:
Constructor based injection is preferred: remove Autowire, add "final" and on class level use Lombok's @RequiredArgsConstructor